src/nupic/engine/__init__.py

```diff
@@ -49,7 +49,7 @@
     ("nupic.regions.KNNClassifierRegion", "KNNClassifierRegion"),
     ("nupic.regions.PluggableEncoderSensor", "PluggableEncoderSensor"),
     ("nupic.regions.PyRegion", "PyRegion"),
-    ("nupic.regions.RecordSensor", "RecordSensor"),
```

Comment:
@vitaly-krugl @scottpurdy Here I have updated the region registration, but still getting a test error:

```
> python tests/unit/nupic/regions/record_sensor_region_test.py
```

```
ERR:  CHECK FAILED: "pClass && PyType_Check(pClass)"  [/Users/mtaylor/nta/nupic.core/src/nupic/py_support/PyHelpers.cpp line 656]
WARN:   PyRegion::createSpec failed: 0x7fff521cddd8
EERR:  CHECK FAILED: "pClass && PyType_Check(pClass)"  [/Users/mtaylor/nta/nupic.core/src/nupic/py_support/PyHelpers.cpp line 656]
WARN:   PyRegion::createSpec failed: 0x7fff521cddd8
EERR:  CHECK FAILED: "pClass && PyType_Check(pClass)"  [/Users/mtaylor/nta/nupic.core/src/nupic/py_support/PyHelpers.cpp line 656]
WARN:   PyRegion::createSpec failed: 0x7fff521cdf48
E
======================================================================
ERROR: testActValueOut (__main__.RecordSensorRegionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit/nupic/regions/record_sensor_region_test.py", line 118, in testActValueOut
    network = _createNetwork()
  File "tests/unit/nupic/regions/record_sensor_region_test.py", line 36, in _createNetwork
    network.addRegion('sensor', 'py.record_sensor', '{}')
  File "/Users/mtaylor/nta/nupic/src/nupic/engine/__init__.py", line 641, in addRegion
    engine_internal.Network.addRegion(self, name, nodeType, nodeParams)
  File "/Users/mtaylor/nta/nupic.core/bindings/py/src/nupic/bindings/engine_internal.py", line 1167, in addRegion
    return _engine_internal.Network_addRegion(self, *args, **kwargs)
AttributeError: 'module' object has no attribute 'record_sensor'

======================================================================
ERROR: testBucketIdxOut (__main__.RecordSensorRegionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit/nupic/regions/record_sensor_region_test.py", line 111, in testBucketIdxOut
    network = _createNetwork()
  File "tests/unit/nupic/regions/record_sensor_region_test.py", line 36, in _createNetwork
    network.addRegion('sensor', 'py.record_sensor', '{}')
  File "/Users/mtaylor/nta/nupic/src/nupic/engine/__init__.py", line 641, in addRegion
    engine_internal.Network.addRegion(self, name, nodeType, nodeParams)
  File "/Users/mtaylor/nta/nupic.core/bindings/py/src/nupic/bindings/engine_internal.py", line 1167, in addRegion
    return _engine_internal.Network_addRegion(self, *args, **kwargs)
AttributeError: 'module' object has no attribute 'record_sensor'

======================================================================
ERROR: testVaryingNumberOfCategories (__main__.RecordSensorRegionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit/nupic/regions/record_sensor_region_test.py", line 72, in testVaryingNumberOfCategories
    "sensor", "py.record_sensor", "{'numCategories': 2}")
  File "/Users/mtaylor/nta/nupic/src/nupic/engine/__init__.py", line 641, in addRegion
    engine_internal.Network.addRegion(self, name, nodeType, nodeParams)
  File "/Users/mtaylor/nta/nupic.core/bindings/py/src/nupic/bindings/engine_internal.py", line 1167, in addRegion
    return _engine_internal.Network_addRegion(self, *args, **kwargs)
AttributeError: 'module' object has no attribute 'record_sensor'

----------------------------------------------------------------------
Ran 3 tests in 0.005s

FAILED (errors=3)
```